### ISSUE TITLE
Data processing necessary?

### ISSUE THREAD

**Issue by Spaceface16518** [Spaceface16518/strat-collector#20]
Do we really need to include the data processor module? We could always just send the data straight from reddit to mongodb.

**Comment by vorana** [comment]
Yes we need it. We want to format it BEFORE it goes into the database. We need to run the formatting script before we send it to the Database to achieve this. Inquire for more info, ill be starting work on this in free time.
